mongo/integration/mtest/sent_message.go

```diff
@@ -46,6 +48,69 @@ func getSentMessageParser(opcode wiremessage.OpCode) (sentMsgParseFn, bool) {
 	}
 }
 
+func parseOpQuery(wm []byte) (*SentMessage, error) {
```

Comment:
This was removed here: https://github.com/mongodb/mongo-go-driver/commit/44496173f8f03c24842511790fb99df4badf901b#diff-f2f0224660237c189dbf0df8f484d0c7165bfd5f2541ab66b18fa822c054cdf9

It needs to be added back to be able to read and evaluate legacy hello commands in the integration tests.